nats-base-client/types.ts

```diff
@@ -943,6 +948,7 @@ export interface KvOptions extends KvLimits {
   streamName: string;
   codec: KvCodecs;
   bindOnly: boolean;
+  allow_direct: boolean;
```

Comment:
This is for all streams, although agree it will be more prevalent with KV.

Reply:
👍